server/routes/v1/deployment.ts

```diff
@@ -89,9 +92,13 @@ export const postDeployment = [
       owner: req.user!._id,
     })
 
-    req.log.info('Saving deployment model')
+    req.log.info({ deployment }, 'Saving deployment model')
     await deployment.save()
 
+    req.log.info(
+      { modelId: model._id, version: body.highLevelDetails.initialVersionRequested },
```

Comment:
`modelId: model._id` should be `model` here.

Reply:
Changed